core/domain/exp_services.py

```diff
@@ -70,6 +70,9 @@
 # search query.
 MAX_ITERATIONS = 10
 
+# Asset directories that should not be bundled for download.
+DOWNLOAD_RESTRICTED_ASSET_DIRS = ['audio']
```

Comment:
Let's name this more clearly: ASSET_DIRS_TO_EXCLUDE_FROM_DOWNLOADS

Though actually -- I think it would be better to whitelist instead. Could we instead do ASSET_DIRS_TO_INCLUDE_IN_DOWNLOADS? That would have better default behaviour, IMO.